tests/net/automatic_testing/README.rst

```diff
@@ -0,0 +1,88 @@
+.. _network-automatic-testing:
+
+Network Automatic Testing
+#########################
+
+Overview
+********
+
+This test application for Zephyr will setup two virtual LAN networks
+and provides echo-server service for normal and encrypted UDP and
+TCP connections. The test application also enables net-shell.
+
+The source code for this test application can be found at:
+:file:`tests/net/automatic_testing`.
+
+Requirements
+************
+
+- :ref:`networking_with_qemu`
+
+Building and Running
+********************
+
+Normally this test application is launched by automatic test system.
+It is also possible to run this testing application with QEMU as described in
+:ref:`networking_with_qemu`, with native-posix board or with real hardware.
+Note that VLAN is only supported for boards that have ethernet port.
+
+Follow these steps to build the testing application:
+
+.. zephyr-app-commands::
+   :zephyr-app: tests/net/automatic_testing
+   :board: <board to use>
+   :conf: prj.conf
+   :goals: build
+   :compact:
+
+If this application is run in native_posix board, then normally one needs extra
+priviliges to create and configure the TAP device in the host system.
+Use sudo command to execute the Zephyr process with admin privileges, like this:
+
+.. code-block:: console
+
+    sudo --preserve-env=ZEPHYR_BASE make run
+
+If the sudo command reports an error, then try to execute it like this:
+
+.. code-block:: console
+
+    sudo --preserve-env make run
+
+The default configuration file prj.conf creates two virtual LAN networks
+with these settings and one normal ethernet interface:
+
+- VLAN tag 100: IPv4 198.51.100.1 and IPv6 2001:db8:100::1
+- VLAN tag 200: IPv4 203.0.113.1 and IPv6 2001:db8:200::1
+- IPv4 192.0.2.1 and IPv6 2001:db8::1
+
+Setting up Linux Host
+=====================
+
+The :file:`samples/net/vlan/vlan-setup-linux.sh` provides a script that can be
+executed on the Linux host. It creates two VLAN on the Linux host and
+suitable IP routes to Zephyr. This script is not needed for native-posix board
```

Comment:
`` for a native-posix board as the host IP address...``  (adding ``a`` and ``the``)